src/components/input/baseInput/baseInput.tsx

```diff
@@ -70,6 +68,7 @@ function BaseInputElement(
   const [isEmojiMenuShown, setIsEmojiMenuShown] = useState(false)
   const inputRef = useRef<HTMLInputElement>(null)
 
+  const database = new Database({ dataSource: props.emojiDataSource })
```

Comment:
`props.emojiDataSource` is optional but isnt the expected type string here ?

Reply:
The emoji library always gives it a default value.